### PR TITLE
Update silicon-labs-vcp-driver

### PR DESCRIPTION
**Pull request by isometry**
* Update download and homepage URLs
* Add version (4.10.15), checksum and appcast checkpoint

After making all changes to the cask:

- [x] `brew cask audit --download silicon-labs-vcp-driver` is error-free.
- [x] `brew cask style --fix silicon-labs-vcp-driver` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/silicon-labs-vcp-driver.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 cask 'silicon-labs-vcp-driver' do
-  version :latest
-  sha256 :no_check
+  version '4.10.15'
```
Comment (vitorgalvao) [inline Casks/silicon-labs-vcp-driver.rb]: This needs to stay as it was, or the cask will break with every update.

Comment (isometry) [inline Casks/silicon-labs-vcp-driver.rb]: That was the point… If we don't include a version number, how is the brew-cask machinery ever meant to detect that the version the user has installed is out-of-date?
With my patch as proposed, the appcast checkpoint changing will indicate that the cask needs to be updated, the version can be updated, following which the brew-cask machinery can respond appropriately to `brew upgrade`. One or two users may indeed hit an error before the cask is updated, but I naively assumed there was automated process to detect appcast checkpoint changes and allow cask updates before users hit problems?

Comment (vitorgalvao) [inline Casks/silicon-labs-vcp-driver.rb]: > how is the brew-cask machinery ever meant to detect that the version the user has installed is out-of-date?

We don’t have upgrade functionality yet.

Missed you added the appcast. It’s fine in that case. However, you seem to greatly misunderstand how its supposed to work. `appcast` is for internal use of tools that help us detect outdated casks.

Comment (isometry) [inline Casks/silicon-labs-vcp-driver.rb]: That matches my understanding of appcast, though my wording may have been poor.

**Pull request merged**